docs/src/snap/reference/annotations.md

```diff
@@ -1,35 +1,169 @@
+
+|   |   |
+|---|---|
+|**Values**| “IPIP”\|”VXLAN”\|”IPIPCrossSubnet”\|”VXLANCrossSubnet”\|”None”|
+|**Description**|The type of encapsulation to use on the IPv4 pool.|
+
+## `k8sd/v1alpha1/calico/encapsulation-v6`
+
+|   |   |
+|---|---|
+|**Values**| “IPIP”\|”VXLAN”\|”IPIPCrossSubnet”\|”VXLANCrossSubnet”\|”None”|
+|**Description**|The type of encapsulation to use on the IPv6 pool.|
+
+## `k8sd/v1alpha1/calico/autodetection-v4/firstFound`
+
+|   |   |
+|---|---|
+|**Values**| "true"\|"false"|
+|**Description**|Use default interface matching parameters to select an interface, performing best-effort filtering based on well-known interface names.|
+
+## `k8sd/v1alpha1/calico/autodetection-v4/kubernetes`
+
+|   |   |
+|---|---|
+|**Values**| “NodeInternalIP”|
+|**Description**|Configure Calico to detect node addresses based on the Kubernetes API.|
+
+## `k8sd/v1alpha1/calico/autodetection-v4/interface`
+
+|   |   |
+|---|---|
+|**Values**| string |
+|**Description**|Enable IP auto-detection based on interfaces that match the given regex.|
+
+## `k8sd/v1alpha1/calico/autodetection-v4/skipInterface`
+
+|   |   |
+|---|---|
+|**Values**| string |
+|**Description**|Enable IP auto-detection based on interfaces that do not match the given regex.|
+
+## `k8sd/v1alpha1/calico/autodetection-v4/canReach`
+
+|   |   |
+|---|---|
+|**Values**| string |
+|**Description**|Enable IP auto-detection based on which source address on the node is used to reach the specified IP or domain.|
+
+## `k8sd/v1alpha1/calico/autodetection-v4/cidrs`
+
+|   |   |
+|---|---|
+|**Values**| []string (comma separated) |
+|**Description**|Enable IP auto-detection based on which addresses on the nodes are within one of the provided CIDRs.|
+
+## `k8sd/v1alpha1/calico/autodetection-v6/firstFound` 
+
+|   |   |
+|---|---|
+|**Values**| "true"\|"false" |
+|**Description**|Use default interface matching parameters to select an interface, performing best-effort filtering based on well-known interface names.|
+
+## `k8sd/v1alpha1/calico/autodetection-v6/kubernetes`
+
+|   |   |
+|---|---|
+|**Values**| “NodeInternalIP” |
+|**Description**|Configure Calico to detect node addresses based on the Kubernetes API.|
+
+## `k8sd/v1alpha1/calico/autodetection-v6/interface`
+
+|   |   |
+|---|---|
+|**Values**| string |
+|**Description**|Enable IP auto-detection based on interfaces that match the given regex.|
+
+## `k8sd/v1alpha1/calico/autodetection-v6/skipInterface`
+
+|   |   |
+|---|---|
+|**Values**| string |
+|**Description**|Enable IP auto-detection based on interfaces that do not match the given regex.|
+
+## `k8sd/v1alpha1/calico/autodetection-v6/canReach` 
+
+|   |   |
+|---|---|
+|**Values**| string |
+|**Description**|Enable IP auto-detection based on which source address on the node is used to reach the specified IP or domain.|
+
+## `k8sd/v1alpha1/calico/autodetection-v6/cidrs`
+
+|   |   |
+|---|---|
+|**Values**| []string (comma separated) |
```

Comment:
the description here is inconsistent. I get that []string means something to golang, but we aren't expecting all readers to be golang experts. We should probably settle on a single way of describing this. suggestions?